composer.json

```diff
@@ -7,7 +7,7 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": ">=5.5",
-        "google/auth": "^1.2.0",
+        "google/auth": "dev-mtls as 1.13.0",
```

Comment:
Before we merge, this needs to be updated to require the latest tagged version of `google/auth`